internal/init/init.go

```diff
@@ -85,40 +86,31 @@ func (c *Client) NewCommand() *cobra.Command {
 			fmt.Println()
 			token := string(b)
 			c.Servicer.SetToken(token)
-			metalClient := c.Servicer.API(cmd)
-			c.UserService = metalClient.Users
-			c.ProjectService = metalClient.Projects
+			metalGoClient := c.Servicer.MetalAPI(cmd)
+			c.UserService = *metalGoClient.UsersApi
+			c.ProjectService = *metalGoClient.ProjectsApi
 
-			user, _, err := c.UserService.Current()
+			user, _, err := c.UserService.FindCurrentUser(context.Background()).Execute()
 			if err != nil {
 				return err
 			}
-			organization := user.DefaultOrganizationID
-			project := ""
-			if user.DefaultProjectID != nil {
-				project = *user.DefaultProjectID
-			}
+			organization := user.AdditionalProperties["default_organization_id"]
+
 			fmt.Printf("Organization ID [%s]: ", organization)
 
 			userOrg := ""
 			fmt.Scanln(&userOrg)
 			if userOrg == "" {
-				userOrg = organization
+				userOrg = fmt.Sprintf("%v", organization)
 			}
 
-			// Choose the first project in the preferred org
-			if project == "" {
-				project, err = getFirstProjectID(c.ProjectService, userOrg)
```

Comment:
I don't think, I feel at least one project should be default at any case, anyway we can get default Org ID using Metal-go

Reply:
The user will have a default org, but the code that is being removed here looks for an alternative project ID if the user doesn't have a default project ID, and I think we need to keep that so that the init command behavior stays the same

Reply:
yes, agree. we don't have relation ship  API to get Org ID from projects, We have API to get projects form Org ID